test/e2etest.py

```diff
@@ -2636,6 +2637,52 @@ def test_twilio_delivery_update(fake_message_id):
     assert re.json()['twilio_delivery_status'] == 'delivered'
 
 
+def test_twilio_retry(fake_message_id):
+    if not fake_message_id:
+        pytest.skip('We do not have enough data in DB to do this test')
+
+    message_sid = uuid.uuid4().hex
+
+    with iris_ctl.db_from_config(sample_db_config) as (conn, cursor):
+        cursor.execute('''DELETE FROM `twilio_delivery_status` WHERE `message_id` = %s''', fake_message_id)
+        cursor.execute('''INSERT INTO `twilio_delivery_status` (`twilio_sid`, `message_id`)
+                          VALUES (%s, %s)''', (message_sid, fake_message_id))
+        cursor.execute('''DELETE FROM `twilio_retry` WHERE `retry_id` = %s OR `message_id` = %s''',
```

Comment:
you could do `%(id)s` and `{'id': fake_message_id}` to avoid specifying it twice.